packages/blade/src/components/SkipNav/__tests__/SkipNav.web.test.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import userEvents from '@testing-library/user-event';
+import renderWithTheme from '../../../_helpers/testing/renderWithTheme.web';
+import { SkipNavContent, SkipNavLink } from '../SkipNav.web';
+
+describe('<SkipNav />', () => {
+  it('<SkipNavLink /> should render correctly', () => {
+    const { getByRole } = renderWithTheme(<SkipNavLink />);
+
+    expect(getByRole('link')).toHaveTextContent('Skip to content');
+  });
+
+  it('<SkipNavLink /> should render correctly with children', () => {
+    const { getByRole } = renderWithTheme(<SkipNavLink children="Skip" />);
+
+    expect(getByRole('link')).toHaveTextContent('Skip');
+  });
+
+  it('<SkipNavContent /> should render correctly', () => {
+    const { getByTestId } = renderWithTheme(<SkipNavContent />);
+
+    expect(getByTestId('skipnav-content')).toHaveAttribute('tabindex', '-1');
```

Comment:
will this by default have `tabindex: -1` ?

Reply:
Yes. TabIndex -1 so that the element isn't focusable through keyboard but can be focused by other means (mouse, .focus etc)